tests/Validators/PriceValidatorTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Tests\Validators;
+
+use App\Validators\PriceValidator;
+use Tests\TestCase;
+
+class PriceValidatorTest extends TestCase
+{
+    public function testValidatePriceSuccess()
+    {
+        $price = '12345';
+        $result = PriceValidator::validatePrice($price);
+        $this->assertEquals('12345', $result);
+    }
+
+    public function testValidatePriceFailure()
+    {
+        $price = 'ABCD';
+        $this->expectException(\Exception::class);     
```

Comment:
This should be more specific for our case. This just expects any exception but we are expecting our specific custom exception.